src/DataSource/Http.hs

```diff
@@ -37,7 +40,7 @@ defaultConfig =
         { cfgAgentId            = 0
         , cfgZendesk            = "https://iohk.zendesk.com"
         , cfgToken              = ""
-        , cfgEmail              = "daedalus-bug-reports@iohk.io"
+        , cfgEmail              = "kristijan.saric@iohk.io"
```

Comment:
You should revert this to `daedalus-bug-reports@iohk.io`. This address is used when the classifier is posting comments on Zendesk.

Reply:
We need to move some of these to CLI, this doesn't make sense here.